.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+daysUntilStale: 90
+daysUntilClose: 90
+
+exemptLabels:
+- type/bug
+- type/cleanup
+- type/feature
+- type/infrastructure
```

Comment:
As discussed in yesterday's meeting: Since we're adding type labels to pretty much all issues, this would exempt almost all of them.

Let's remove "cleanup", "feature" and "infrastructure" and add "regression" instead.

Reply:
We can also exempt issues that are in projects, milestones, or assigned.

```yaml
# Set to true to ignore issues in a project (defaults to false)
exemptProjects: false

# Set to true to ignore issues in a milestone (defaults to false)
exemptMilestones: false

# Set to true to ignore issues with an assignee (defaults to false)
exemptAssignees: false
```

Reply:
Do we perhaps also want a label that makes sure the stale bot won't close it? One that doesn't fit those labels but that we want to keep open? We have a bunch of those I think, and making updates every 90 days so they aren't closed or marked as stale might just harm us instead of helping us.

Reply:
assigned, milestone and project can all be set to true I think

Reply:
@datitisev Maybe a `org/keep` tag to indicate that it's not a type of ticket but rather an organizational thing

Reply:
I think that's a very fair suggestion, but would actually refrain from adding it until we determine it really is needed. Once we need it, your naming suggestion sounds nice. :+1: